nodes/src/main/java/io/aexp/nodes/graphql/annotations/GraphQLProperty.java

```diff
@@ -25,4 +25,5 @@
 
     String name();
     GraphQLArgument[] arguments() default {};
+    GraphQLVariable[] variables() default {};
```

Comment:
Because variables are now definable w/ @GraphQLProperty -- they are definable at class level now through GraphQLProperty only. Do you want to update @GraphQLVariable(s) annotation to class level as well?

Reply:
Yep! I was going to add it here first and then another PR for both the GraphQLArgument and GraphQLVariable to be definable at the class level.
Just wanted to make this PR first for all the logic to handle a class level definition of it

Reply:
Okay.